client/python/cryoet_data_portal/src/cryoet_data_portal/_gql_base.py

```diff
@@ -135,7 +138,7 @@ def get_related_class(self):
         return self.__current_query.get_related_class()
 
     def to_gql(self):
-        return self.__name
+        return strcase.to_lower_camel(self.__name)
```

Comment:
This seems to cause an exception in `strcase`, as `self.__name` is a list, not a string. 

to reproduce: 
```python
import cryoet_data_portal as cdp

client = cdp.Client()
point_annos = cdp.AnnotationFile.find(
    client,
    [
        cdp.AnnotationFile.tomogram_voxel_spacing.run_id == 1000,
        cdp.AnnotationFile.annotation_shape.shape_type._in(["Point", "OrientedPoint"]),  # noqa
    ],
)
```

`TypeError: expected string or bytes-like object`

Reply:
Fix seems to be as easy as 

```python
return [strcase.to_lower_camel(n) for n in self.__name]
```